### ISSUE TITLE
'toByteData' isn't defined for the class 'Future<Image>'

### ISSUE THREAD

**Issue by stevenspiel** [4Q-s-r-o/signature#2]
This is a breaking change as noted [here](https://github.com/flutter/flutter/issues/26655)

Here is the error message and backtrace:

```
[VERBOSE-2:ui_dart_state.cc(148)] Unhandled Exception: 'package:signature/signature.dart': error: file:///Users/steven/flutter/.pub-cache/hosted/pub.dartlang.org/signature-1.0.3/lib/signature.dart:201:29: Error: The method 'toByteData' isn't defined for the class 'Future<Image>'.
 - 'Future' is from 'dart:async'.
 - 'Image' is from 'dart:ui'.
Try correcting the name to the name of an existing method, or defining a method named 'toByteData'.
    var bytes = await image.toByteData(format: ImageByteFormat.png);
                            ^^^^^^^^^^
#0      _AsyncAwaitCompleter.start (dart:async/runtime/libasync_patch.dart:49:6)
#1      _SignaturePainter.export (package:signature/signature.dart:187:27)
#2      SignatureState.exportBytes (package:signature/signature.dart:65:26)
<asynchronous suspension>
#3      Signature.exportBytes (package:signature/signature.dart:34:35)
<asynchronous suspension>
#4      Drawable.build.<anonymous closure> (package:zboviq/pages/shared/drawable.dart:81:74)
#5      _InkRe<…>
[VERBOSE-2:ui_dart_state.cc(148)] Unhandled Exception: 'package:signature/signature.dart': error: file:///Users/steven/flutter/.pub-cache/hosted/pub.dartlang.org/signature-1.0.3/lib/signature.dart:201:29: Error: The method 'toByteData' isn't defined for the class 'Future<Image>'.
 - 'Future' is from 'dart:async'.
 - 'Image' is from 'dart:ui'.
Try correcting the name to the name of an existing method, or defining a method named 'toByteData'.
    var bytes = await image.toByteData(format: ImageByteFormat.png);
                            ^^^^^^^^^^
#0      _AsyncAwaitCompleter.start (dart:async/runtime/libasync_patch.dart:49:6)
#1      _SignaturePainter.export (package:signature/signature.dart:187:27)
#2      SignatureState.exportBytes (package:signature/signature.dart:65:26)
<asynchronous suspension>
#3      Signature.exportBytes (package:signature/signature.dart:34:35)
<asynchronous suspension>
#4      Drawable.build.<anonymous closure> (package:zboviq/pages/shared/drawable.dart:81:74)
#5      _InkRe<…>
```

**Comment by stevenspiel** [comment]
[This is how another person solved the issue](https://github.com/lukef/qr.flutter/pull/17/files)

**Comment by ethael** [comment]
hello. thanks a lot. we will resolve this problem this week

**Comment by MartinHlavna** [comment]
Released as version 1.1.0

**Comment by MSatyam-Mishra** [comment]
is it not fixed yet? coz I am getting this error " The method 'toByteData' isn't defined for the type 'Future'."
